app/src/main/java/org/standardnotes/notes/MainActivity.kt

```diff
@@ -25,6 +26,8 @@ class MainActivity : AppCompatActivity() {
         fab.setOnClickListener { view ->
             (supportFragmentManager.findFragmentById(R.id.noteListFrag) as NoteListFragment).startNewNote()
         }
+
+        SyncManager.startSyncTimer()
```

Comment:
Ah yeah good call.  I had kind of assumed it wouldn't matter that the 30s timer is also running in NoteActivity, but then you'd have a race condition where edits wouldn't get synced when typing while the global timer was syncing..